en/identity-server/next/docs/deploy/performance/configure-cache-layers.md

```diff
@@ -3,11 +3,11 @@
 WSO2 Identity Server has multiple cache layers which are used to improve performance. The following configurations that can be configured in the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory are used to manage and configure the cache layers.
```

Comment:
Not directly related to your changes, but we are fixing the vale lint errors in the changed files as part of each PR. Shall we update this text as follows to fix the vale lint errors?

```suggestion
WSO2 Identity Server includes several cache layers that boost performance. You can manage and configure the cache layers using the following settings in the `deployment.toml` file located in the `<IS_HOME>/repository/conf/` directory.
```